readme.uk.md

```diff
@@ -9,10 +9,10 @@ Mate Royal Oil вирішили повністю автоматизувати п
 
 ```js
 customer = {
-  money: 3000, // залишок грошей на рахунку клієнта
+  money: 3000,
   vehicle {
```

Comment:
There is a syntax error in the `customer` object definition. The `vehicle` property should be defined with a colon, like `vehicle: {`.